packages/components/src/internal/components/EditInlineField.tsx

```diff
@@ -139,28 +168,35 @@ export const EditInlineField: FC<Props> = memo(props => {
                         autoFocus
                         className="form-control"
                         cols={50}
-                        defaultValue={value}
+                        defaultValue={_value}
                         onBlur={onBlur}
                         onFocus={onTextAreaFocus}
                         onKeyDown={onKeyDown}
                         name={name}
+                        placeholder={placeholder}
                         ref={inputRef}
                         rows={5}
                     />
                 </span>
             )}
-            {state.editing && isText && (
+            {state.editing && column && !isDate && (
+                <Formsy className="form-horizontal" onChange={onFormsyColumnChange}>
```

Comment:
BTW, I'm seeing some javascript console errors that might be related to this (form within a form).  Noted in my testing notes.